website/docs/style/icon/icons-list.js

```diff
@@ -4305,6 +4340,60 @@ const iconsList = {
         'лукер',
       ],
     },
+    {
+      name: 'Mailchimp',
+      size: ['l', 'm'],
+      group: 'Color',
+      tags: [
+        'logo',
+        'логотип',
+        'лого',
+        'mailchimp',
+        'мейлчимп',
+        'обезьянка',
+        'monkey',
+        'почта',
+        'рассылка',
+        'email',
+        'маркетинг',
+        'marketing',
+      ],
+    },
+    {
+      name: 'MatomoColored',
+      size: ['l', 'm'],
+      group: 'Color',
+      tags: [
+        'logo',
+        'логотип',
+        'лого',
+        'matomo',
+        'матомо',
+        'веб',
+        'web',
+        'analytics',
+        'аналитика',
+        'software',
+        'софт',
+      ],
+    },
+    {
+      name: 'Microsoft',
+      size: ['l', 'm'],
+      group: 'Color',
+      tags: [
+        'microsoft',
+        'ms',
+        'micro',
+        'soft',
+        'office',
+        'микрософт',
+        'мелкософт',
```

Comment:
is it appropriate? :)
```suggestion
        'мелкософт',
```

Reply:
I'll remove these old guys 😅